File: backend/Entities/Models/FilialLastMileGrupo.cs
using System;
using System.Collections.Generic;

namespace Entities.Models;

public partial class FilialLastMileGrupo : BaseEntity
{
    public Guid Id { get; set; }

    public string Nome { get; set; } = null!;

    public string? Codigo { get; set; }

    public int? Ordem { get; set; }

    public bool? Ativo { get; set; }

    public virtual ICollection<FilialLastMileGrupoItem> FilialLastMileGrupoItems { get; } = new List<FilialLastMileGrupoItem>();
}

File: frontend/src/app/Business/Models/filialLastMileGrupo.ts
import { FilialLastMileGrupoItem } from "./filialLastMileGrupoItem";

export interface FilialLastMileGrupo  {
    id: string;
    nome: string;
    codigo: string | null;
    ordem: number | null;
    ativo: boolean | null;
    filialLastMileGrupoItems: FilialLastMileGrupoItem[];
}
